xsd/siri_model/siri_situationActions.xsd

```diff
@@ -509,7 +509,7 @@ Rail transport, Roads and road transport
      </xsd:element>
      <xsd:element name="UserRef" type="xsd:string" default="true" minOccurs="0">
```

Comment:
"true" is a strange default for a user reference! No default makes more sense.

Reply:
done